pom.xml

```diff
@@ -36,7 +36,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.10.5.1</version>
+            <version>2.13.5</version>
```

Comment:
can we use the same strategy of updating parent?

Reply:
I mean updating the jackson version in parent, then updating the parent?
https://github.com/confluentinc/common/blob/master/pom.xml#L60

Reply:
done

Reply:
oh, its quite different from other connectors, it doesn't have the parent.